### PR TITLE
NIFI-3291: Upgrade jQuery and plugins

### PR DESCRIPTION
**Pull request by mcgilman**
- Upgrading jQuery and it's plugins.
- Packaging at build time using npm.

### CODE REVIEW

**Comment by scottyaslan** [comment]
Will review...

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/pom.xml [hunk 1]
```diff
@@ -710,8 +784,6 @@
                         <configuration>
                             <packagingIncludes>
                                 assets/**/*,
-                                css/reset.css,
-                                css/reset.css.gz,
                                 css/common-ui.css,
                                 css/common-ui.css.gz,
                                 css/message-page.css,
```
Comment (scottyaslan) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/pom.xml]: We need to add the slick-nifi-theme here:

css/slick-nifi-theme.css,
css/slick-nifi-theme.css.gz,

**Comment by scottyaslan** [comment]
I also noticed there are scrollbars showing up in the "Access Policies" shell as well as the "Add Reporting Task" dialog.

policies scrollbar issue

rt scrollbar issue

**Comment by mcgilman** [comment]
Thanks. Just updated the PR.

**Comment by scottyaslan** [comment]
Thanks @mcgilman this has been merged to master.

**Pull request closed without merging**